packages/engine/src/scene/systems/SceneObjectSystem.tsx

```diff
@@ -57,16 +58,17 @@ export function setupObject(obj: Object3DWithEntity) {
   const mesh = obj as any as Mesh<any, any>
   mesh.traverse((child: Mesh<any, any>) => {
     if (child.material) {
-      if (isHMD && ExpensiveMaterials.has(child.material.constructor)) {
+      if (!Engine.instance.isEditor && !isHMD && ExpensiveMaterials.has(child.material.constructor)) {
```

Comment:
incorrect inversion of isHMD (should be `isHMD` not `!isHMD`)

Reply:
good catch